src/test/java/io/confluent/connect/elasticsearch/MappingTest.java

```diff
@@ -75,6 +75,24 @@ public void testInferMapping() throws Exception {
     assertEquals(0, intMapping.get("properties").get("foo").get("null_value").asInt());
   }
 
+  @Test
+  public void testInferMappingDefaultDate()  {
+    java.util.Date expected = new java.util.Date();
+
+    Schema dateSchema = SchemaBuilder
+        .struct()
+        .name("record")
+        .field("foo", Date.builder().defaultValue(expected).build())
+        .build();
+
+    JsonNode dateMapping = Mapping.inferMapping(client, dateSchema);
+    assertNotNull(dateMapping.get("properties").get("foo").get("null_value"));
+    assertEquals(
+        expected.getTime(),
+        dateMapping.get("properties").get("foo").get("null_value").asLong()
+    );
```

Comment:
add a test for long default values too?

Reply:
so default value for a Date can never be a Long - SchemaBuilder will throw an exception...

Reply:
hmm, I meant to test these two lines: https://github.com/confluentinc/kafka-connect-elasticsearch/pull/135/files/e925cebfd15c8503639a1ec38a5a1fefec20ce7e#diff-9525ecd31fdc68633378d58447bea186R212-R213. if they can never be triggered, then we should not have those lines.

Reply:
just future proofing :)